tests/hook_main.py

```diff
@@ -901,38 +904,41 @@ def invoice_patch(transaction):
 @hooks.before("Invoices > Event Invoices Details > Delete Event Invoices")
 def invoice_delete(transaction):
     """
-    DELETE /event-invoices/1
+    DELETE /event-invoices/b0929ea6
     :param transaction:
     :return:
     """
     with stash['app'].app_context():
         event_invoice = EventInvoiceFactory()
+        event_invoice.identifier = "b0929ea6"
         db.session.add(event_invoice)
         db.session.commit()
 
 
 @hooks.before("Invoices > Event Invoice List of an Event > List Event Invoices of an Event")
 def event_event_invoice_get_list(transaction):
     """
-    GET /events/1/event-invoices
+    GET /events/b0929ea6/event-invoices
```

Comment:
-2 sense

Reply:
Forgot to revert it while pushing. Fixing it now